controllers/managedocs_controller.go

```diff
@@ -151,6 +156,7 @@ type ManagedOCSReconciler struct {
 	addonParams                        map[string]string
 	onboardingValidationKeySecret      *corev1.Secret
 	prometheusKubeRBACConfigMap        *corev1.ConfigMap
+	rhobsSecret                        *corev1.Secret
```

Comment:
Lets have the variable name capture the full context 
```suggestion
	rhobsRemoteWriteConfigSecret                        *corev1.Secret
```

Reply:
Done